src/utility.c

```diff
@@ -342,18 +345,9 @@ int strn_cmp(char *arg1, char *arg2, int n) {
 
 /* writes a string to the log */
 void log_sev(char *str, int sev) {
```

Comment:
Given that this no longer writes to `stderr`, and only to the logged-in wizards, should this get renamed now? (at least, it needs a new comment!)